src/solvers/dgsem_tree/dg_2d_parallel.jl

```diff
@@ -11,12 +11,12 @@
 
 # TODO: MPI dimension agnostic
 # TODO: MPI, adapt to different real types (and AD!)
-mutable struct MPICache
+mutable struct MPICache{uEltype}
```

Comment:
```suggestion
mutable struct MPICache{uEltype <: Real}
```
That's something we enforce for the other containers, too.